lib/presentation/bookPages/bloc/book_bloc.dart

```diff
@@ -162,10 +186,25 @@ class BookBloc extends Bloc<BookEvents, BookState> {
       await FirebaseDbManager.instance
           .updatePage(book.bookData!.pages, book.id);
       currentPageIndex = book.bookData!.pages.indexOf(event.bookPage);
-
       emit(LoadedBookPageState());
-      emit(DisplayBookPageState(
-          bookData: book.bookData!, pageIndex: currentPageIndex));
+
+      Map<String, dynamic> additionalData = {
+        'action': pageAddedAction,
+        'bookId': book.id,
+        'index': currentPageIndex.toString()
+      };
+
+      final result = await FCM.instance.sendPushMessage(
+        topic: messageTopic,
+        title: 'New page has been edited',
+        body:
```

Comment:
Maybe the title and body text could be sent through event so that it could be localised. Send localised text through event and then use it  here.